frontend/src/components/pages/MultiUserBurndownChart/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, {useState} from "react";
+import Presentation from "./Presentation";
+import { connect } from "react-redux";
+import { fillInSnapshotDateGaps } from "./utils";
+
+export function MultiUserBurndownChartUnconnected({
+  //   mapStateToProps
+  currentTeamBurndownStats,
+  team,
+  metrics,
+  // mapDispatchToProps
+
+  // storybook
+}) {
+  const [metricFilter, setMetricFilter] = useState("projectCardsTotalCount");
+
+  const handleChangeMetricFilter = (newValue) => {
+    setMetricFilter(newValue);
+  };
+  
+  currentTeamBurndownStats = currentTeamBurndownStats
+    ? Object.values(currentTeamBurndownStats)
```

Comment:
This looks like a bug. You are saying that if it exists, then make an Array, otherwise initialize it to an empty object. The moment we connect this to real data it will break.